parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -244,7 +245,7 @@ try
     # Download CSE function scripts
     Write-Log "Getting CSE scripts"
     $tempfile = 'c:\csescripts.zip'
-    DownloadFileOverHttp -Url $global:CSEScriptsPackageUrl -DestinationPath $tempfile -ExitCode $global:WINDOWS_CSE_ERROR_DOWNLOAD_CSE_PACKAGE
+    Logs-To-Events "AKS.WindowsCSE.DownloadCSEScriptPackageUrl" DownloadFileOverHttp -Url $global:CSEScriptsPackageUrl -DestinationPath $tempfile -ExitCode $global:WINDOWS_CSE_ERROR_DOWNLOAD_CSE_PACKAGE

```

Comment:
It seems like that we can log the values inside the Windows node in node provisioning to the kusto with this method, k8s url, containerd url, cse url, etc. for troubleshooting.

Reply:
Thanks for the comment. Added the related logs.